src/lib.rs

```diff
@@ -337,8 +337,6 @@ pub struct SerialPortBuilder {
 
 impl SerialPortBuilder {
     /// Set the path to the serial port
-    // TODO: Switch to `clone_into` when bumping our MSRV past 1.63 and remove this exemption.
```

Comment:
This doesn't appear to address the *"Switch to `clone_into`"* part of this TODO.

Reply:
Fixed. I decided to use `into_owned` instead of `clone_into`, since `clone_into` copies the memory if `Cow::Owned`, while `into_owned` doesn't.